ykrt/try_passes.py

```diff
@@ -62,10 +59,10 @@ def get_all_passes(is_prelink):
     seen = set()
      
     for descr in pass_descrs:
-        parts = descr.split("<")
-        if parts[0] not in seen:
-            seen.add(parts[0])
-            passes.append(Pass(parts[0]))
+        if descr not in seen:
+            seen.add(descr)
+            passes.append(Pass(descr))
+    
```

Comment:
So we simply pass through the default arguments to each pass?

It wasn't what I expected, but it's probably a decent first step.

(I thought that we might try different sets of arguments, but let's not do that right now)

Reply:
oh, i have a question: where do we get these new arguments from if not from default?